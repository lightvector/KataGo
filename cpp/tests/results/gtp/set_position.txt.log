: GTP Engine starting...
: KataGo v1.8.2
: Using TrompTaylor rules initially, unless GTP/GUI overrides this
: Using 1 CPU thread(s) for search
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 19 nnYLen 19
: Loaded config configs/gtp_example.cfg and/or command-line and query overrides
: Loaded model tests/models/g170-b6c96-s175395328-d26788732.bin.gz
: Config override: cudaUseFP16 = false
: Config override: cudaUseNHWC = false
: Config override: forDeterministicTesting = true
: Config override: logDir = 
: Config override: logFile = tests/results/gtp/set_position.txt.log
: Config override: logTimeStamp = false
: Config override: maxPlayouts = 10000
: Config override: maxVisits = 100
: Config override: nnRandSeed = forTesting
: Config override: nnRandomize = false
: Config override: numSearchThreads = 1
: Config override: openclUseFP16 = false
: Config override: searchRandSeed = forTesting
: Model name: g170-b6c96-s175395328-d26788732
: GTP ready, beginning main protocol loop
: Controller: showboard
: = MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b c3 W q16 black q17 BLACK k10 black c5 WHITE q13
: = 
: Controller: showboard
: = MoveNum: 0 HASH: A927404D59D756B072DFBA7052A05BF5
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . X . . .
16 . . . . . . . . . . . . . . . O . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . O . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . X . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . X . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . X . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position
: = 
: Controller: showboard
: = MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: boardsize 13
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 13 nnYLen 13
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 6D7B9BD0653F4FE761D8EA5338E563E0
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . . . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position w d3 w d10 w j10
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 3BFA3DC657A6A5EFD2BEA7778146E5BE
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . O . . . . O . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d3 b d10 b j10
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d3 b d10 b j10 b
: ? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'b d3 b d10 b j10 b'
: Controller: showboard
: = MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position w d3 w    d10  w  j10
: ? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'w d3 w    d10  w  j10'
: Controller: showboard
: = MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d3 b d3
: ? Illegal stone placements - overlapping stones or stones with no liberties?
: Controller: showboard
: = MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d3 w d3
: ? Illegal stone placements - overlapping stones or stones with no liberties?
: Controller: showboard
: = MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d4 w d3 w e4 w d5
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b d4 w d3 w e4 w d5 w c4
: ? Illegal stone placements - overlapping stones or stones with no liberties?
: Controller: showboard
: = MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b pass
: ? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'b pass': Could not parse vertex: 'pass'
: Controller: showboard
: = MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: boardsize 3
: GPU -1 finishing, processed 0 rows 0 batches
: Cleaned up old neural net and bot
: nnRandSeed0 = forTesting
: After dedups: nnModelFile0 = tests/models/g170-b6c96-s175395328-d26788732.bin.gz useFP16 false useNHWC false
: Cuda backend thread 0: Found GPU Tesla V100-SXM2-16GB memory 16945512448 compute capability major 7 minor 0
: Cuda backend thread 0: Model version 8 useFP16 = false useNHWC = false
: Cuda backend thread 0: Model name: g170-b6c96-s175395328-d26788732
: Loaded neural net with nnXLen 7 nnYLen 7
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 710F257116C1B48A7B9876BE820C87F2
   A B C
 3 . . .
 2 . . .
 1 . . .
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b a1 b a2 b a3 b b1 b b2 b b3 b c1 b c2
: = 
: Controller: showboard
: = MoveNum: 0 HASH: 3EF6B2A37C09369B375BF765A930F411
   A B C
 3 X X .
 2 X X X
 1 X X X
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: Controller: set_position b a1 b a2 b a3 b b1 b b2 b b3 b c1 b c2 b c3
: ? Illegal stone placements - overlapping stones or stones with no liberties?
: Controller: showboard
: = MoveNum: 0 HASH: 3EF6B2A37C09369B375BF765A930F411
   A B C
 3 X X .
 2 X X X
 1 X X X
Next player: Black
Rules: {"friendlyPassOk":false,"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0
: GPU -1 finishing, processed 0 rows 0 batches
: All cleaned up, quitting
