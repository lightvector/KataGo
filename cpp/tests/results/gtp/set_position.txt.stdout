= MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: A927404D59D756B072DFBA7052A05BF5
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . X . . .
16 . . . . . . . . . . . . . . . O . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . O . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . X . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . X . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . X . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: CDCBC1F514D7E680FACD226074256633
   A B C D E F G H J K L M N O P Q R S T
19 . . . . . . . . . . . . . . . . . . .
18 . . . . . . . . . . . . . . . . . . .
17 . . . . . . . . . . . . . . . . . . .
16 . . . . . . . . . . . . . . . . . . .
15 . . . . . . . . . . . . . . . . . . .
14 . . . . . . . . . . . . . . . . . . .
13 . . . . . . . . . . . . . . . . . . .
12 . . . . . . . . . . . . . . . . . . .
11 . . . . . . . . . . . . . . . . . . .
10 . . . . . . . . . . . . . . . . . . .
 9 . . . . . . . . . . . . . . . . . . .
 8 . . . . . . . . . . . . . . . . . . .
 7 . . . . . . . . . . . . . . . . . . .
 6 . . . . . . . . . . . . . . . . . . .
 5 . . . . . . . . . . . . . . . . . . .
 4 . . . . . . . . . . . . . . . . . . .
 3 . . . . . . . . . . . . . . . . . . .
 2 . . . . . . . . . . . . . . . . . . .
 1 . . . . . . . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 6D7B9BD0653F4FE761D8EA5338E563E0
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . . . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 3BFA3DC657A6A5EFD2BEA7778146E5BE
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . O . . . . O . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'b d3 b d10 b j10 b'

= MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'w d3 w    d10  w  j10'

= MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Illegal stone placements - overlapping stones or stones with no liberties?

= MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Illegal stone placements - overlapping stones or stones with no liberties?

= MoveNum: 0 HASH: 5F4FCF10528E2A9D9AAD58BBDBB6C452
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . X . . . . X . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . . . . . . . . . . .
 4 . . . . . . . . . . . . .
 3 . . . X . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Illegal stone placements - overlapping stones or stones with no liberties?

= MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Expected a space-separated sequence of <COLOR> <VERTEX> pairs but got 'b pass': Could not parse vertex: 'pass'

= MoveNum: 0 HASH: 0060C2712F9F96DF15BC0200D6BF3DFD
   A B C D E F G H J K L M N
13 . . . . . . . . . . . . .
12 . . . . . . . . . . . . .
11 . . . . . . . . . . . . .
10 . . . . . . . . . . . . .
 9 . . . . . . . . . . . . .
 8 . . . . . . . . . . . . .
 7 . . . . . . . . . . . . .
 6 . . . . . . . . . . . . .
 5 . . . O . . . . . . . . .
 4 . . . X O . . . . . . . .
 3 . . . O . . . . . . . . .
 2 . . . . . . . . . . . . .
 1 . . . . . . . . . . . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 710F257116C1B48A7B9876BE820C87F2
   A B C
 3 . . .
 2 . . .
 1 . . .
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

= 

= MoveNum: 0 HASH: 3EF6B2A37C09369B375BF765A930F411
   A B C
 3 X X .
 2 X X X
 1 X X X
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

? Illegal stone placements - overlapping stones or stones with no liberties?

= MoveNum: 0 HASH: 3EF6B2A37C09369B375BF765A930F411
   A B C
 3 X X .
 2 X X X
 1 X X X
Next player: Black
Rules: {"hasButton":false,"ko":"POSITIONAL","komi":7.5,"scoring":"AREA","suicide":true,"tax":"NONE","whiteHandicapBonus":"0"}
B stones captured: 0
W stones captured: 0

